### ISSUE TITLE
Allow to load add ccViewer in Files app

### ISSUE THREAD

**Issue by prabirshrestha** [lithium0003/ccViewer#21]
This added support for accessing files from ccViewer. https://github.com/lithium0003/ccViewer/issues/2. I was expecting the opposite instead i.e. from Files app I can view files and folders from ccViewer. Sorry if I wasn't clear. I didn't think of #2 but that seems also a good feature that I will be using.

This allows me to easily use `Save to Files` from a different app and select ccViewer. Files app also supports compress and uncompress so features like those can be got for free.

**Comment by lithium0003** [comment]
It means that showing the ccViewer folders when open the Files.app? My understand is that you want to import (and upload) files from other app with Files.app, is it correct?

**Comment by prabirshrestha** [comment]
Yes. Basically I want ccViewer to be shown here in the Files app location. This would allow me to easily import export from other apps but use ccViewer for encryption.

![E21CC432-9DBF-45C8-92E4-AC6EBFB98F1A](https://user-images.githubusercontent.com/287744/70024359-aa7a5e00-154e-11ea-8c63-c8540c2c9eda.png)

**Comment by lithium0003** [comment]
Thanks for information. It means that app should implement FileProvider. I'll check it and try.

**Comment by lithium0003** [comment]
#4
